.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+  schedule:
+    - cron: '0 0 * * sun,wed'
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: Linux, python: 3.6, os: ubuntu-16.04, tox: 'flake8,flake8-docs'}
+          - {name: Linux, python: 3.6, os: ubuntu-16.04, tox: 'py36-linux'}
+          - {name: Linux, python: 3.7, os: ubuntu-16.04, tox: 'py37-linux'}
+          - {name: Linux, python: 3.8, os: ubuntu-18.04, tox: 'py38-linux'}
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup up Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Update pip
+        run: python -m pip install -U pip
+      - name: Install tox, gdb
+        run: |
+          pip install tox
+          sudo apt-get install -y gdb
+      - name: Enable core dumps
+        run: ulimit -c unlimited -S  # enable core dumps
+      - name: Run tox tests
+        run: tox -e ${{ matrix.tox }} -vv
```

Comment:
Is vv necessary here? Tox will spew lots of output that nobody really ever reads.

Reply:
I'm following the existing tests in .travis.yml. I can remove it.